cdap-watchdog-api/src/test/java/io/cdap/cdap/api/metrics/MetricValueTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.cdap.api.metrics;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class MetricValueTest {
+    @Test
+    public void testGetAllCounts() {
+
+        long bucketMask = 1 /* first position */ +
+                4 /* 2^2 i.e. third position */ +
```

Comment:
You can use shift to get nicer constants (e.g. `1 << 2` for 4)

Reply:
Good suggestion. Changed it.